apps/mark/backend/src/app.ts

```diff
@@ -47,42 +45,41 @@ function buildApi(
 
     // TODO: Once election definition has been moved to the backend, no longer require the frontend
     // to provide election hash to this and other methods that use the auth lib
```

Comment:
Delete this comment?

Reply:
Good catch! https://github.com/votingworks/vxsuite/pull/3357/commits/74d6b15c20b323502aa988053ad2b9e115b8f22f